docker-compose-services/typo3-solr/README.md

```diff
@@ -0,0 +1,48 @@
+## TYPO3-specific Apache Solr Integration for DDEV-Local
+
+Although ddev has [documented generic Solr support](https://ddev.readthedocs.io/en/stable/users/extend/additional-services/#apache-solr) it is as simple as possible, and supports only a single core named "dev". 
+
+The TYPO3 extension assumes a different approach and a slightly different Solr image. 
+
+These instructions were tested with TYPO3 LTS v9.5.
+
+Resources:
+* ApacheSolrForTypo3 EXT:Solr [docs (master)](https://docs.typo3.org/p/apache-solr-for-typo3/solr/master/en-us/)
+* [typo3solr/ext-solr](https://hub.docker.com/r/typo3solr/ext-solr/) Solr image on hub.docker.com.
+* [typo3solr Slack Channel](https://typo3.slack.com/messages/ext-solr/)  (request your invite for TYPO3 Slack at https://forger.typo3.org/slack)
+* [Original Stack Overflow Tutorial](https://stackoverflow.com/questions/51479399/how-to-set-up-solr-server-for-typo3-using-ddev) that this is based on.
+
+1. Add the solr extension to your project: `ddev composer require apache-solr-for-typo3/solr`
+2. In the "Extensions" module deactivate and then re-activate the "Apache Solr for TYPO3" module to make sure that its database tables get installed.
+3. Copy [docker-compose.solr.yaml](docker-compose.solr.yaml) to your project's .ddev folder.
+4. If you want your solr data to be persistent across `ddev restart`, then uncomment the `- solrdata:/var/solr` line in docker-compose.solr.yaml. The comments there explain what you have to do if you want to start over. It's recommended to wait to uncomment that until you have everything else working.
+5. Copy the default Solr configuration from Ext:Solr to ddev:
+    * `mkdir -p .ddev/solr`
+    * `cp -r public/typo3conf/ext/solr/Resources/Private/Solr/* .ddev/solr`
+    * You will have `configsets`, `cores`, `solr.xml` and `zoo.cfg` in .ddev/solr.
+6. `ddev start` will bring up the new solr container.
+   * Click the "Apache Solr" "Info" module on the left and choose your site, you should see that it has connected the Apache Solr server.
```

Comment:
Shows me the message `There is no site configured for your solr extension!`. So this should be moved after site setup point 7

Reply:
I did change the ordering. Hope you'll try again..

Reply:
Works perfect now! Had no more issues following exactly your guide